src/test/java/seedu/address/logic/parser/RateCommandParserTest.java

```diff
@@ -0,0 +1,43 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_RATING;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseFailure;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseSuccess;
+import static seedu.address.testutil.TypicalIndexes.INDEX_FIRST_BOOK;
+
+import org.junit.Test;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.RateCommand;
+import seedu.address.model.book.Rating;
+
+public class RateCommandParserTest {
+    private RateCommandParser parser = new RateCommandParser();
+    private final int nonEmptyRating = 5;
```

Comment:
`NON_EMPTY_RATING`

Reply:
add another constant `EMPTY_RATING` = -1. Replace all uses of -1 with this constant.